plugin/root/README.md

```diff
@@ -2,14 +2,15 @@
 
 ## Name
 
-*root* - simply specifies the root of where to find (zone) files.
+*root* - simply specifies the root of where to find config files.
```

Comment:
I don't think all plugins check root, so we cant claim exactly this yet. Perhaps we need to list plugins that do, or add root check or plugins that don't.

Reply:
I would not explicitly list the plugins, rather I would try to make it consistent and fix it for all the plugins

Reply:
So what other files could there be?

What I found so far:
* Kubernetes kubeconfig
* dnssec key file (this might already respect the root path?)
* forward
* sign plugin (also seems to already respect the root path)

What else comes to your mind?

Reply:
Check also for indirect file access, where a path or file name is passed to an imported package. Eg cloudns and tsig secrets, and others.